### PR TITLE
Expose only needed error codes

### PR DESCRIPTION
**Pull request by renkelvin**
Expose only needed error codes
- [New] FIRAuthErrorCodeRecaptchaNotEnabled
- [Existing and reused] FIRAuthErrorCodeCaptchaCheckFailed

### CODE REVIEW

File: FirebaseAuth/Tests/Sample/Podfile [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
+source "https://github.com/CocoaPods/Specs.git"
 
 use_frameworks!
 
 target 'AuthSample' do
-  platform :ios, '11.0'
+  platform :ios, '12.0'
   pod 'FirebaseCore', :path => '../../../'
   pod 'FirebaseCoreInternal', :path => '../../../'
-  pod 'FirebaseAuth', :path => '../../../', :testspecs => ['unit']
```
Comment (paulb777) [inline FirebaseAuth/Tests/Sample/Podfile:15]: This works if you set the environment variable expected by the podspec first.

Comment (renkelvin) [reply FirebaseAuth/Tests/Sample/Podfile:15]: Yeah, thanks for pointing it out.

**Comment by renkelvin** [comment]
Gonna split this PR into smaller pieces.

**Review by paulb777 — APPROVED** [review]
Looks like the Firebase project got corrupted and is causing an integration test failure.

Please fix and then LGTM

**Comment by renkelvin** [comment]
> Looks like the Firebase project got corrupted and is causing an integration test failure.
> 
> Please fix and then LGTM

Yeah, fixed the testing project.

**Pull request merged**